modules/terraform/options.go

```diff
@@ -71,6 +71,8 @@ type Options struct {
 	PlanFilePath             string                 // The path to output a plan file to (for the plan command) or read one from (for the apply command)
 	PluginDir                string                 // The path of downloaded plugins to pass to the terraform init command (-plugin-dir)
 	SetVarsAfterVarFiles     bool                   // Pass -var options after -var-file options to Terraform commands
+	AdditionalInitFlags      []string               // additional flags to pass to init command - e.g. `-backend=false`. complete list of options [here](https://developer.hashicorp.com/terraform/cli/commands/init)
+	// AdditionalApplyDestroylags []string               // additional flags to pass to apply/destroy command
```

Comment:
`AdditionalApplyDestroylags` is required? - looks like it is commented and not referenced

Reply:
Yes correct ...  can be deleted.